core/src/main/java/org/helsinki/paytrail/request/payments/PaytrailPaymentCreateMitChargeRequest.java

```diff
@@ -0,0 +1,49 @@
+package org.helsinki.paytrail.request.payments;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.experimental.Accessors;
+import org.helsinki.paytrail.PaytrailClient;
+import org.helsinki.paytrail.request.common.PaytrailPostRequest;
+import org.helsinki.paytrail.response.payments.PaytrailPaymentCreateMitChargeResponse;
+
+import java.util.TreeMap;
+
+public class PaytrailPaymentCreateMitChargeRequest extends PaytrailPostRequest<PaytrailPaymentCreateMitChargeResponse> {
+    private final CreateMitChargePayload payload;
+
+    public PaytrailPaymentCreateMitChargeRequest(
+            @NonNull CreateMitChargePayload payload
+    ) {
+        this.setCheckoutTransactionId(payload.getToken());
+        this.payload = payload;
+    }
+
+    @Override
+    protected CreateMitChargePayload getPayload(PaytrailClient client) {
+        return payload;
+    }
+
+    @Override
+    protected TreeMap<String, String> getRequestSpecificHeaders() {
+        return null;
```

Comment:
Should transactionId be setted to headers?
https://docs.paytrail.com/#/?id=create-authorization-hold-or-charge 

from paytrail docs:
Note! The transaction id needs to be sent on checkout-transaction-id header as well.

Reply:
But which transactionId? We don't receive a transactionId anywhere in the previous calls. We only receive it upon requesting the mit charge. In the ticket, it's been written that checkout-transaction-id = token. Now, I don't know what's the correct way.

Reply:
This was resolved, token is added to headers in applyDefaultHeaders if checkout-transaction-id is not null :D My bad